lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -1,3 +1,4 @@
+from http import HTTPStatus
```

Comment:
This line should be positioned after the bare `import`'s and before the project `import`'s.